subsys/bluetooth/mesh/sar_cfg_internal.h

```diff
@@ -11,9 +11,9 @@
 #ifndef ZEPHYR_SUBSYS_BLUETOOTH_MESH_SAR_CFG_INTERNAL_H__
 #define ZEPHYR_SUBSYS_BLUETOOTH_MESH_SAR_CFG_INTERNAL_H__
 
-/** SAR Transmitter Configuraion state encoded length */
+/** SAR Transmitter Configuration state encoded length */
 #define BT_MESH_SAR_TX_LEN 4
-/** SAR Receiver Configuraion state encoded length */
+/** SAR Receiver Configuration state encoded length */
```

Comment:
Since you are touching this, `Configuration` can be removed as in spec they are just `SAR Transmitter` and `SAR Receiver` states.